src/commons/api/history.rs

```diff
@@ -538,144 +493,167 @@ impl WithStorableDetails for StorableCaCommand {
+            CertAuthStorableCommand::AspasUpdate { .. } => CommandSummary::new("cmd-ca-aspas-update", self),
+            CertAuthStorableCommand::AspasUpdateExisting { .. } => {
+                CommandSummary::new("cmd-ca-aspas-update-existing", self)
+            }
+            CertAuthStorableCommand::AspaRemove { .. } => CommandSummary::new("cmd-ca-aspas-remove", self),
 
             // BGPSec
-            StorableCaCommand::BgpSecDefinitionUpdates => CommandSummary::new("cmd-bgpsec-update", self),
+            CertAuthStorableCommand::BgpSecDefinitionUpdates => CommandSummary::new("cmd-bgpsec-update", self),
 
             // REPO
-            StorableCaCommand::RepoUpdate { service_uri } => {
+            CertAuthStorableCommand::RepoUpdate { service_uri } => {
                 CommandSummary::new("cmd-ca-repo-update", self).with_service_uri(service_uri)
             }
 
-            StorableCaCommand::ReissueBeforeExpiring => CommandSummary::new("cmd-ca-reissue-before-expiring", self),
-            StorableCaCommand::ForceReissue => CommandSummary::new("cmd-ca-force-reissue", self),
+            CertAuthStorableCommand::ReissueBeforeExpiring => {
+                CommandSummary::new("cmd-ca-reissue-before-expiring", self)
+            }
+            CertAuthStorableCommand::ForceReissue => CommandSummary::new("cmd-ca-force-reissue", self),
 
             // RTA
-            StorableCaCommand::RtaPrepare { name } => {
+            CertAuthStorableCommand::RtaPrepare { name } => {
                 CommandSummary::new("cmd-ca-rta-prepare", self).with_rta_name(name)
             }
-            StorableCaCommand::RtaSign { name } => CommandSummary::new("cmd-ca-rta-sign", self).with_rta_name(name),
-            StorableCaCommand::RtaCoSign { name } => CommandSummary::new("cmd-ca-rta-cosign", self).with_rta_name(name),
+            CertAuthStorableCommand::RtaSign { name } => {
+                CommandSummary::new("cmd-ca-rta-sign", self).with_rta_name(name)
+            }
+            CertAuthStorableCommand::RtaCoSign { name } => {
+                CommandSummary::new("cmd-ca-rta-cosign", self).with_rta_name(name)
+            }
 
             // Deactivation
-            StorableCaCommand::Deactivate => CommandSummary::new("cmd-ca-deactivate", self),
+            CertAuthStorableCommand::Deactivate => CommandSummary::new("cmd-ca-deactivate", self),
         }
     }
+
+    fn make_init() -> Self {
+        Self::Initialise
+    }
 }
 
-impl fmt::Display for StorableCaCommand {
+impl fmt::Display for CertAuthStorableCommand {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
+            // ------------------------------------------------------------
+            // Initialisation
+            // ------------------------------------------------------------
+            CertAuthStorableCommand::Initialise => write!(f, "Create CA"),
```

Comment:
Stupid question: Is initialisation really the same thing as "creating" a CA? Initialisation isn't an internal detail that can happen more than once for a CA or is separate to a CA?

Reply:
Also for the equivalent below you wrote:
```
            StorableRepositoryCommand::Initialise => {
                write!(f, "Initialise server")
            }
```
So why write "Initialise" there and "Create" here?

Reply:
Yeah, you're right. As with "version" and "sequence", I should choose one name for this thing. I will go for "Initialise".